standalone/firepit.js

```diff
@@ -972,19 +971,6 @@ function FindTool(bin) {
     .filter(p => p);
 }
 
-function SetWindowTitle(title) {
-  /*
-    This method *attempts* to set the terminal window title to something pretty so it doesn't
-    show the internal shell'ing we do. It kinda works, but fails silently, so I've left it in.
-   */
-  if (isWindows) {
-    process.title = title;
-  } else {
```

Comment:
Can we actually leave the `isWindows` branch of this code (which causes no problems) and remove the `else` branch?

Reply:
Sure ♻️ https://github.com/firebase/firebase-tools/pull/2330/commits